CRM/Core/BAO/MessageTemplate.php

```diff
@@ -379,12 +381,16 @@ public static function sendTemplate($params) {
       }
     }
 
-    $subject = $dao->subject;
-    $text = $dao->text;
-    $html = $dao->html;
-    $format = $dao->format;
+    $mailContent = array(
+      'subject' => $dao->subject,
+      'text' => $dao->text,
+      'html' => $dao->html,
+      'format' => $dao->format,
+    );
     $dao->free();
 
+    CRM_Utils_Hook::alterMailContent($mailContent);
+
```

Comment:
But is it better to take this array and turn it into separate variables?  I know it would minimize this diff here, but I feel it would do so at the expense of clarity regarding what's actually going on.  In other words, we need `$mailContent['subject']` in order to use the hook; is there really an advantage to turning that into `$subject` later?